include/eve/module/core/named_shuffles/swap_adjacent.hpp

```diff
@@ -65,52 +65,52 @@ struct swap_adjacent_t
   }
 
   template<simd_value T, std::ptrdiff_t G>
-  static constexpr std::ptrdiff_t level(eve::as<T> tgt, eve::fixed<G> g)
```

Comment:
these tests in `level` were deliberately not if constexpr where posssible. 
main consideration is that if constexpr is probably slower to compile.
it also, for example, does not to do else branches.